app/shared/src/forms/TextInput.js

```diff
@@ -36,26 +102,17 @@ const styles = StyleSheet.create({
     marginLeft: 10,
     alignSelf: 'center',
   },
+  placeholder: {
+    position: 'absolute',
+    padding: 10,
+    paddingVertical: 15,
+    // this should be the same with 'input' above
+    color: Color.textLight,
```

Comment:
I don't understand this comment, color should be the same as input above? But it is not, it is textLight and input has textDark

Reply:
This should be resolved now. Please check again.